tests/testTriggers.go

```diff
@@ -573,18 +610,48 @@ func CollectEventRecords(recordChan *chan *EventRecord) {
 // TriggerEmailReporter sends email with all reported errors
 func TriggerEmailReporter(contexts []*scheduler.Context, recordChan *chan *EventRecord) {
 	// emailRecords stores events to be notified
-	emailRecords := emailRecords{}
+
+	emailData := emailData{}
 	logrus.Infof("Generating email report: %s", time.Now().Format(time.RFC1123))
+
+	var masterNodeList []string
+	var pxStatus string
+
+	for _, n := range node.GetMasterNodes() {
+		masterNodeList = append(masterNodeList, n.Addresses...)
+	}
+	emailData.MasterIP = masterNodeList
+
+	for _, n := range node.GetWorkerNodes() {
+		status, err := Inst().V.GetNodeStatus(n)
+		if err != nil {
+			pxStatus = "ERROR GETTING STATUS"
+
+		}
+
+		if *status == api.Status_STATUS_OK {
+			pxStatus = "OPERATIONAL"
+		} else {
+			pxStatus = "NON-OPERATIONAL"
```

Comment:
Instead of categorizing PX status in either `OPERATIONAL` or `NON-OPERATIONAL` buckets, let's just dump whatever is the status. That way, we will have more visibility into whats the exact state PX is in, on that node.

Reply:
Addressed these changes